apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/r4/providers/HospiceClaimTransformerV2Test.java

```diff
@@ -261,13 +266,18 @@ public void shouldHaveExtensions() {
     hospiceCountExtension.setValue(new Quantity(2));
 
     List<Extension> compare =
-        Arrays.asList(
+        List.of(
             new Extension(
                 "https://bluebutton.cms.gov/resources/variables/nch_near_line_rec_ident_cd",
                 new Coding(
                     "https://bluebutton.cms.gov/resources/variables/nch_near_line_rec_ident_cd",
                     "V",
                     "Part A institutional claim record (inpatient [IP], skilled nursing facility [SNF], hospice [HOS], or home health agency [HHA])")),
+            new Extension(
+                "https://bluebutton.cms.gov/resources/variables/fi_doc_clm_cntl_num",
+                new Identifier()
+                    .setSystem("https://bluebutton.cms.gov/resources/variables/fi_doc_clm_cntl_num")
```

Comment:
can we reference the constant here?

Reply:
I typically refrain from referencing program code constants as it would mask errors related to typos in the original constant definition.

Reply:
thats an interesting take and looking at the rest of that test it can certainly be argued that you are following the patterns there (although they are not 100% consistent within the test) so I'm not going to push the issue although I would argue that we should verify those constants in one place only (possibly relying on the actual response verification tests as the golden standard) and then use the references in all of the other tests so as to make the code easier to maintain if for some reason they should need to change later.